tasks/18.html

```diff
@@ -12,7 +12,20 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
-                /* your styles here */
+                article:hover div
+                {
+                    opacity: .25;
+                }
+
+                div
+                {
+                    opacity: .5;
+                }
+
+                div:hover
```

Comment:
Можно обойтись без `!important`. Достаточно указать родительский тег в селекторе:
```suggestion
                article div:hover
```

Можешь почитать про вычисление специфичности здесь: https://developer.mozilla.org/ru/docs/Web/CSS/Specificity